pkgs/servers/mail/opensmtpd/filter-rspamd.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "opensmtpd-filter-rspamd";
+  version = "0.1.7";
+
+  src = fetchFromGitHub {
+    owner = "poolpOrg";
+    repo = "filter-rspamd";
+    rev = "v${version}";
+    sha256 = "1qhrw20q9y44ffgx5k14nvqc9dh47ihywgzza84g0zv9xgif7hd5";
+  };
+
+  vendorSha256 = "0wp87lziry8x4gd94qbqk1sd2admrbyh790kl75cn55nxmrpdldh";
+
+  meta = with lib; {
+    homepage = "https://github.com/poolpOrg/filter-rspamd";
+    description = "OpenSMTPD filter integration for the Rspamd daemon";
+    license = licenses.isc;
```

Comment:
Missing maintainer.

Reply:
Who should I add?
I'd prefer not to commit myself to maintaining this package (although, since I'm using it, if I do find an issue that affects me then I would do my best to fix it).

Reply:
Maybe we should be merging https://github.com/NixOS/nixpkgs/pull/122823 then, which was just sent and appears to add the author there as a maintainer?